builder-function.Dockerfile.j2

```diff
@@ -17,4 +17,4 @@ COPY Makefile .
 # and the app itself
 COPY tsconfig.json ./
 COPY src src
-COPY backend backend
+#COPY backend backend
```

Comment:
You can remove this line, `backend` is the destination directory for `tsc` in this case (pending my PR)

Reply:
@michaeladda already removed in #3